IMC_DataScience/IMC_OOPTutorial/blog.md

```diff
@@ -29,7 +29,9 @@ In 1969, Martin M. Broadwell made one of the first references to a psychological
 
 
 
-This guide hopes to be different. I’m assuming, if you’re reading this, you’re on your journey from incompetence to competence, and to this end have the fundamentals already down - variables, loops, basic syntax, and other things like that. These lessons will discuss higher level concepts and techniques, and hopefully convince you that the past few hours you spent on Codecademy or whatever your teaching method of choice was wasn’t a waste of time. Even making it to stage 3 in this context - competence with coding - is no simple task, and one that we all are taking steps to achieve, but with any luck these articles can help guide you down that path.
+This guide hopes to be different. I’m assuming, if you’re reading this, you’re on your journey from incompetence to competence, and to this end have the fundamentals already down - variables, loops, basic syntax, and other things like that (if not, give [Codecademy](https://www.codecademy.com/learn/learn-java) or some other coding website a visit, and go through the first few lessons they have. You probably don't need to go through the whole thing, but the more the merrier). These lessons will discuss higher level concepts and techniques, and hopefully convince you that the past few hours you spent on Codecademy or whatever your teaching method of choice was wasn’t a waste of time. Even making it to stage 3 in this context - competence with coding - is no simple task, and one that we all are taking steps to achieve, but with any luck these articles can help guide you down that path.
```

Comment:
Your writing style is a little too informal and stream of consciousness overall. These aren't supposed to read like textbooks, but using words such as even or later on line 44 obviously don't help. Descriptions such as "other things like that" can be cleaned up to something like "etc." or "and much more".

This is a bit of a rough draft, but feel free to refer to this: https://hackmd.io/ucEy4YP9Sd2iioC3jQi1Hg